src/components/App.jsx

```diff
@@ -180,60 +180,46 @@ export default function App() {
                 </rb.Nav.Item>
               </div>
             )}
-            <div className="d-flex flex-row">
-              <rb.Nav.Item>
-                <a
-                  href="https://github.com/joinmarket-webui/joinmarket-webui/wiki"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  className="nav-link text-secondary px-2"
-                >
-                  {t('footer.docs')}
-                </a>
-              </rb.Nav.Item>
-              <rb.Nav.Item>
-                <a
-                  href="https://github.com/joinmarket-webui/joinmarket-webui#-features"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  className="nav-link text-secondary px-2"
-                >
-                  {t('footer.features')}
-                </a>
-              </rb.Nav.Item>
-              <rb.Nav.Item>
-                <a
-                  href="https://github.com/joinmarket-webui/joinmarket-webui"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  className="nav-link text-secondary px-2"
-                >
-                  {t('footer.github')}
-                </a>
-              </rb.Nav.Item>
-              <rb.Nav.Item>
-                <a
-                  href="https://twitter.com/joinmarket"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  className="nav-link text-secondary px-2"
+          </div>
+          <div className="d-flex flex-1 order-2 justify-content-end align-items-center gap-1">
+            <div className="d-flex gap-2 pe-2">
+              <a
+                href="https://github.com/joinmarket-webui/joinmarket-webui"
+                target="_blank"
+                rel="noopener noreferrer"
+                className="d-flex align-items-center text-secondary"
+              >
+                <Sprite symbol="github" width="18px" height="18px" className="" />
+              </a>
+              <a
+                href="https://t.me/JoinMarketWebUI"
+                target="_blank"
+                rel="noopener noreferrer"
+                className="d-flex align-items-center text-secondary"
+              >
+                <Sprite symbol="telegram" width="18px" height="18px" className="" />
+              </a>
+            </div>
+            <div className="d-flex text-secondary">|</div>
+            <div className="d-flex">
+              <rb.OverlayTrigger
+                delay={{ hide: 300, show: 200 }}
+                placement="top"
+                overlay={(props) => (
+                  <rb.Tooltip {...props}>
+                    {websocketConnected ? t('footer.connected') : t('footer.disconnected')}
+                  </rb.Tooltip>
+                )}
+              >
```

Comment:
Tooltip is nice. :+1:

It is misaligned in my browsers though, but that's an issue with react-bootstrap, so I do not know what to do about it, just wanted to let you know (Not using safari here).
I tired upgrading to v2.3.0, but it is still not fixed. (Possibly related: https://github.com/react-bootstrap/react-bootstrap/issues/6010 but in general there are some open issue regarding Tooltips/Overlays.)

Reply:
Hm interesting, for me it seems fine on Firefox, Chrome and Safari. Or maybe I'm just not seeing the misalignment? 🤔